Observables/Package.swift

```diff
@@ -0,0 +1,31 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "Observables",
+    platforms: [
+        .macOS(.v10_14), .iOS(.v13), .tvOS(.v13)
+    ],
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "Observables",
+            targets: ["Observables"]),
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        // .package(url: /* package url */, from: "1.0.0"),
+    ],
+    targets: [
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages this package depends on.
+        .target(
+            name: "Observables",
+            dependencies: []),
+        .testTarget(
+            name: "ObservablesTests",
+            dependencies: ["Observables"]),
+    ]
```

Comment:
🛑  When I ran all the unit tests, the tests in this package were not included. I think we'd need to add it to the `UnitTests.xctestplan`. 

![image](https://user-images.githubusercontent.com/198826/101654792-2169f280-39fe-11eb-9a45-502a015db554.png)

Reply:
Ooh yeah – done in https://github.com/woocommerce/woocommerce-ios/pull/3305/commits/cc35693657a7a9579346d0d9547453f16ecd76be